packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -752,4 +752,16 @@
     <label>Résumé</label>
     <name>quantic_Summary</name>
   </customLabels>
+  <customLabels>
+    <label>La réponse générée est cachée</label>
```

Comment:
Ca ne serait pas plutôt: "Le générateur de réponse est désactivé" ?

Reply:
Could be yeah, but in reality we are not really disabling it, we are just hiding it, that's why "cachée"  seems  to me to be the right one.